css/components/Card.css

```diff
@@ -0,0 +1,159 @@
+@import "../style.css";
+
+body{
+    display: flex;
+    flex-wrap: wrap;
+}
+
+/* Card with Badge */
+.badge, .dismiss, .text-overlay, .text, .horizontal, .shadow{
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    width: 200px;
+    height: 270px;
+    border: 1px solid var(--badge-offline);
+    background-color: var(--badge-offline);
+    border-radius: 1rem;
+    margin: .5rem;
+}
+
+.badge .far, .dismiss .far, .text-overlay .far, .text .far, .horizontal .far, .shadow .far{
```

Comment:
Suggestion: class name `far` isn't conveving its meaning.

Reply:
If you add explanation or description comments here it would be good.

Reply:
Changed name to badge

Reply:
change class name to badge